x-pack/plugins/upgrade_assistant/server/lib/es_deprecations_status.ts

```diff
@@ -15,85 +15,126 @@ import {
   convertFeaturesToIndicesArray,
 } from './es_system_indices_migration';
 
+export async function getHealthIndicators(
+  dataClient: IScopedClusterClient
+): Promise<EnrichedDeprecationInfo[]> {
+  const healthIndicators = await dataClient.asCurrentUser.healthReport();
+  console.log(
+    'healthIndicators.indicators.shards_capacity:',
+    healthIndicators.indicators.shards_capacity
+  );
+  return [
+    healthIndicators.indicators.disk,
+    // @ts-ignore
+    healthIndicators.indicators.shards_capacity,
```

Comment:
@JoshMock Can you help us get these types added to the client, there are also other types missing further down in this file related to `MigrationDeprecationInfoResponse`